.travis.yml

```diff
@@ -15,6 +15,9 @@ matrix:
     # also test default configuration with Python 2.7
     - python: 2.7
       env: LMOD_VERSION=6.6.3
+    # also test with Python 3.6
+    - python: 3.6
```

Comment:
Doesn't this depend on testing branch 4.x?

Reply:
It does, the Travis config was changed to build against the `4.x` branch of of `easybuild-framework` and `easybuild-easyblocks` when the target branch for the PR is `4.x`, so it's all good?

Reply:
Ah, yes... this is of course the 4.x branch itself ...
This week has not been good for brain...